### DISCUSSION TITLE
Software and software license not visible in Borrow view

### DISCUSSION THREAD

**Discussion by jejmule** [leihs/leihs#1319] [Q&A / Community Support]
Hello,

I start to configure my inventory. 
I manage to add model/items to my inventory. I realized that  if the category is not defined on the model, they do not show in borrow view.
I add software/license to my inventory but I do not find how to make them visible in the borrow view (borrowable set to true). What should be defined to make it visible?

Thank you,
Jérome

**Comment by jejmule** [comment]
Anyone face this issue.
I guess, I am doing something wrong...

**Comment by nimaai** [comment]
it is the case for both ordinary models and software. if they are not assigned to any category then they are not visible on the catalog page in borrow section but are still found when searched after.

this is not nice imho. we will discuss it in team.
